packages/magento-customer/components/SignUpForm/SignUpFormInline.tsx

```diff
@@ -77,14 +77,16 @@ export function SignUpFormInline({
           variant='outlined'
           type='password'
           error={!!formState.errors.confirmPassword || !!error?.message}
-          label='Confirm password'
+          label={<Trans id='Confirm password' />}
           autoComplete='new-password'
           required
           {...muiRegister('confirmPassword', {
             required: true,
             validate: (value) => value === watchPassword,
           })}
-          helperText={!!formState.errors.confirmPassword && 'Passwords should match'}
+          helperText={
+            !!formState.errors.confirmPassword && i18n._(/* i18n */ 'Passwords should match')
+          }
```

Comment:
I think this should work as well:

```suggestion
          helperText={
            !!formState.errors.confirmPassword && <Trans id="Passwords should match" />
          }
```